### PR TITLE
tests/cgroup: check cgroup.max/cgroup.high values

### PR DESCRIPTION
**Pull request by mihalicyn**
https://github.com/canonical/lxd/pull/13192

### CODE REVIEW

**Review by simondeziel — APPROVED** [review]
LGTM but this seemingly random number got me to scratching my head until I converted 512MiB into bytes ;)

Maybe we should use an expression like
```
MEM_LIMIT_MIB=512
lxc config set c1 -c limits.memory="${MEM_LIMIT_MIB}MiB"

...
[ "$(lxc exec c1 -- cat /sys/fs/cgroup/memory.max)" = $((MEM_LIMIT_MIB * 1024 * 1024)) ]
...
```

**Comment by mihalicyn** [comment]
Thanks, fixed!

**Comment by mihalicyn** [comment]
It is correct, that test is failed:
```
++ lxc exec c1 -- cat /sys/fs/cgroup/memory.max
+ '[' 536870912 = 536870912 ']'
++ lxc exec c1 -- cat /sys/fs/cgroup/memory.high
+ '[' 483180544 = max ']'
```

we need to merge https://github.com/canonical/lxd/pull/13192

File: tests/cgroup [hunk 1]
```diff
@@ -106,14 +106,21 @@ fi
 lxc config unset c1 limits.cpu.allowance
 
 echo "==> Testing memory limits"
-lxc config set c1 limits.memory=512MiB
-[ "$(lxc exec c1 -- grep ^MemTotal /proc/meminfo)" = "MemTotal:         524288 kB" ]
+MEM_LIMIT_MIB=512
+lxc config set c1 limits.memory="${MEM_LIMIT_MIB}MiB"
+[ "$(lxc exec c1 -- grep ^MemTotal /proc/meminfo)" = "MemTotal:         $((512 * 1024)) kB" ]
```
Comment (simondeziel) [inline tests/cgroup:111]:
```suggestion
[ "$(lxc exec c1 -- grep ^MemTotal /proc/meminfo)" = "MemTotal:         $((MEM_LIMIT_MIB * 1024)) kB" ]
```

File: tests/cgroup [hunk 2]
```diff
@@ -106,14 +106,21 @@ fi
...
 if [ -e "/sys/fs/cgroup/memory/memory.memsw.limit_in_bytes" ] || [ -e "/sys/fs/cgroup/memory.swap.max" ]; then
-    [ "$(lxc exec c1 -- grep ^SwapTotal /proc/meminfo)" = "SwapTotal:        524288 kB" ]
+    [ "$(lxc exec c1 -- grep ^SwapTotal /proc/meminfo)" = "SwapTotal:        $((512 * 1024)) kB" ]
```
Comment (simondeziel) [inline tests/cgroup:113]:
```suggestion
    [ "$(lxc exec c1 -- grep ^SwapTotal /proc/meminfo)" = "SwapTotal:        $((MEM_LIMIT_MIB * 1024)) kB" ]
```

Comment (mihalicyn) [reply tests/cgroup]: Ugh, sorry. Wrong copy'n'paste. Have fixed!

**Review by simondeziel — APPROVED** [review]

**Review by tomponline — APPROVED** [review]

**Pull request merged**